software/moltres/wiki/input_example.md

```diff
@@ -612,7 +612,7 @@ of the MOOSE framework. Its useful for creating material properties that are
 functions of solution variables and for coupling the dependency of the property
 on the variable back into the Jacobian used for Newton-Raphson. A more in-depth
 description of the material along with its relatives is given
-[here](http://mooseframework.org/wiki/PhysicsModules/PhaseField/DevelopingModels/FunctionMaterials/).
+[here](https://mooseframework.inl.gov/source/materials/DerivativeParsedMaterial.html).
```

Comment:
Not 100% sure on this one, but I think this hyperlink should direct the user to the FunctionMaterials page, not just the page for the DerivativeParsedMaterial. I believe the appropriate page is https://www.mooseframework.org/modules/phase_field/FunctionMaterials.html

Reply:
I wasn't sure either. The paragraph is about the DerivativeParsedMaterial, but the FunctionMaterials page would provide more context. I'll go ahead and use the FunctionMaterials link since it has a link to the DerivativeParsedMaterial page on it.

Reply:
This has been resolved with a8bebda.